proxy-node-translator/src/dist/config.yml

```diff
@@ -16,11 +16,9 @@ proxyNodeMetadataForConnectorNodeUrl: ${PROXY_NODE_METADATA_FOR_CONNECTOR_NODE_U
 connectorNodeIssuerId: ${CONNECTOR_NODE_ISSUER_ID}
 
 keyRetrieverServiceName: ${KEY_RETRIEVER_SERVICE_NAME:-softHSM}
-softHSMLibPath: ${SOFT_HSM_LIB_PATH}
-softHSMSigningKeyPin: ${SOFT_HSM_SIGNING_KEY_PIN}
-softHSMSigningKeyLabel: ${SOFT_HSM_SIGNING_KEY_LABEL}
 
-connectorFacingSigningKeyPair:
+signerConfiguration:
+  type: file
```

Comment:
Should this be set as an environment variable which defaults to file or does this get overridden else where?

Reply:
We'll probably be overriding this whole YAML block in the Helm charts.